### DISCUSSION TITLE
fastqc on mox

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#95] [Q&A] [answered]
Is it possible to run fastqc on mox using files stored on gannet (without downloading those files)? 

Currently the way I've been doing fastqc is downloading files from gannet to my computer and then running them, but since some of my files are too big to download onto the usb i've been using and my computer is slow, it would be cool if I could run fastqc directly using files on gannet without having to download them first. Further, it'd be cool if I could do this on mox since it would be a bit speedier. 

The main issue I see with this is that I have to make fastqc executable from my applications folder and I don't know how that would work on mox.
The other issue is that I simply don't know how to run fastqc using files from a URL without downloading them (or if that is possible)

**Comment by kubu4** [comment, marked as answer]
You'll need to rsync files from Gannet to Mox. Then run fastqc on Mox.

**Reply by meganewing** [reply]
Okay great. Do I need to do anything to make fastqc executable on mox ? 

Right now the command I used to make it executable is ` chmod u+x /Applications/FastQC/fastQC `

And then I run it with this command: 
` /Applications/FastQC/fastQC \
../data/raw/*fastq.gz \
-o ../analyses/0218-fastqc/ ` 

I'm just confused about what the mox equivalent to the ` /Applications/FastQC/fastQC ` would be for both of those commands ?

**Reply by kubu4** [reply]
Excellent question!

All programs on Mox are here:

`/gscratch/srlab/programs`

FastQC is here:

`/gscratch/srlab/programs/fastqc_v0.11.9/fastqc`

This is the equivalent to what you've been using your computer.
